extensions/rich_text_components/Skillreview/directives/oppia-noninteractive-skillreview.directive.ts

```diff
@@ -55,7 +60,7 @@ angular.module('oppia').directive('oppiaNoninteractiveSkillreview', [
             // instance, then open the concept card modal. To determine if the
             // RTE is inside a CKEditor instance, check if the offsetParent
             // element contains the data attribute ckeWidgetId.
-            if (event.currentTarget.offsetParent.dataset.ckeWidgetId) {
+            if (ctrl.canOpenConceptCardModal(event)) {
```

Comment:
Why are we ending the function if we canOpenConceptCardModal?

Reply:
I should probably change the name of this function. The logic of the function decides whether to show the RTE modal or the concept card modal.

Reply:
Or maybe update the logic of the function? (`return !something && !somethingElse`, although you should prefer changing the name if it's easy and simple to understand.)

Reply:
I'll change it to `shouldOpenRTEModal` - that makes more sense.